incubator/hnc/pkg/reconcilers/object.go

```diff
@@ -290,6 +323,7 @@ func (r *ObjectReconciler) syncPropagated(ctx context.Context, log logr.Logger,
 		return write, srcInst
 	}
 
+	r.addToNumPropagated(log, inst.GetNamespace(), inst.GetName())
```

Comment:
Why is this here? Shouldn't we only call this in line 473, below?

Reply:
If there are propagated objects on the apiserver before we install HNC, we will return `ignore` action here and those objects will not be counted by line 473 ([write](https://github.com/kubernetes-sigs/multi-tenancy/blob/ca59e4a66a81d819183188e7d4c49f754302ff03/incubator/hnc/pkg/reconcilers/object.go#L414) method will not be called). 

For example, if we create namespace a and b, set a to be parent of b, add an object in a. The `NumPropagatedObjects` will be 1 as expected. Then if we reinstall (or restart) HNC, `NumPropagatedObjects` will be 0 if we don't have this line.

Added a comment to explain that.

Reply:
sgtm, thanks!